test/e2e/authorization.go

```diff
@@ -711,19 +752,98 @@ var _ = g.Describe("Authorization via admission-controller [RBAC] [Zalando]", fu
 				framework.ExpectNoError(err, "failed to delete cluster role: %s", nonSystemResource.Name)
 			})
 
+			g.It("should allow write access for collaborator resources", func() {
+				err := client.RbacV1().ClusterRoles().Delete(context.Background(), collaboratorResource.Name, metav1.DeleteOptions{DryRun: []string{"All"}})
```

Comment:
`collaboratorResource` is not initialized in the `BeforeEach`. But given that the outer context is "for global resources", it doesn't make sense to have collaborator resources here because we don't have this concept yet. When referring to the collaborator namespace itself, we can just use the string "visibility".